cpp2sky/segment_context.h

```diff
@@ -50,18 +112,22 @@ class CurrentSegmentSpan {
 
   /**
    * Set start time to calculate execution time.
+   * @param set_time To determine whether to set actual time or not.
+   * This value is introduced for unit-test.
    */
-  virtual void setStartTime(int64_t start_time) = 0;
+  virtual void startSpan(bool set_time = true) = 0;
```

Comment:
Do we still support setting explicit timestamp for start and end?

Reply:
That makes sense because opentracing-cpp provides this knob. https://github.com/opentracing/opentracing-cpp/blob/4bb431f7728eaf383a07e86f9754a5b67575dab0/example/tutorial/tutorial-example.cpp#L60-L66